native/src/main/java/io/ballerina/stdlib/oracledb/parameterprocessor/OracleDBResultParameterProcessor.java

```diff
@@ -132,4 +139,125 @@ public BObject getBalStreamResultIterator() {
         }
         return struct;
     }
+
+    @Override
+    public Object processCustomTypeFromResultSet(ResultSet resultSet, int columnIndex,
+                                                 ColumnDefinition columnDefinition)
+            throws ApplicationError, SQLException {
+        int sqlType = columnDefinition.getSqlType();
+        Type ballerinaType = columnDefinition.getBallerinaType();
+        switch (sqlType) {
+            case -104:
```

Comment:
Would it be better to handle these using constants for better readability?

Reply:
added in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/13/commits/950f2228fe9fde68f2865688c03597cabbea3792